tests/db_engine_specs/presto_tests.py

```diff
@@ -490,3 +491,24 @@ def test_presto_expand_data_array(self):
         self.assertEqual(actual_cols, expected_cols)
         self.assertEqual(actual_data, expected_data)
         self.assertEqual(actual_expanded_cols, expected_expanded_cols)
+
+    @staticmethod
+    def test_get_sqla_column_type():
```

Comment:
no need to make it a class method, in pytest you can just do
```
def test_get_sqla_column_type():
  assert True
```

Reply:
Oh, not sure what I was doing there! I'll just tack it on as a regular class method like the other tests to limit confusion. Do we already have a test module that follows pytest best practices that I can mimic going forward?

Reply:
yep @villebro - check 
https://github.com/apache/incubator-superset/blob/master/tests/csv_upload_tests.py out